### ISSUE TITLE
Documentation for Time Distributed Layer

### ISSUE THREAD

**Issue by vignesh0710** [tensorflow/tensorflow#48209]
Documentation for ```Time Distributed layer``` in Keras mentions any ```Layer``` applied with it will be ```applied to every temporal slice``` 

But in my assumption - for example consider ```mant-to-many``` sequence model trained for ```NER``` or ```Language Model``` with the following code, ```(if return_sequences==True)``` in the previous ```BiLSTM layer``` then ```Dense(n)``` and ```TimeDistributed(Dense(n))```  are exactly the same and either of them can be used. Is my assumption correct? 
   
```
model = Sequential()
model.add(Embedding(input_dim=voc_size, output_dim=embed_dim, input_length=50))
model.add(Bidirectional(LSTM(units=lstm_units,return_sequences=True),
                                         merge_mode="ave"))
model.add(TimeDistributed(Dense(n)))
```

**Comment by Saduf2019** [comment]
@vignesh0710 
Kindly open a [stackoverflow](https://stackoverflow.com/questions/tagged/tensorflow) issue for this as it is not a bug or feature request, Please post this kind of support questions at Stackoverflow. There is a big community to support and learn from your questions.
Thanks!

**Comment by vignesh0710** [comment]
Hi, Thanks for the response. My thinking was in case the assumption is correct, the docs can be modified to assert that as I see so many posts about this very confusion.

**Comment by rmothukuru** [comment]
Created [a CL](https://critique-ng.corp.google.com/cl/367437963) with this change. Thanks!

**Comment by vignesh0710** [comment]
@rmothukuru can you please confirm if the initially mentioned ```assertion``` is correct or not?

**Comment by rmothukuru** [comment]
@vignesh0710,
Yes, the assertion is correct. Please find [this comment from Francois](https://github.com/keras-team/keras/issues/1029#issuecomment-157786333). Thanks!

**Comment by vignesh0710** [comment]
@rmothukuru thanks. Can I close this issue thread then?

**Comment by rmothukuru** [comment]
@vignesh0710,
No, please. The issue will be closed automatically once the CL is successfully submitted.
